scss/_patterns_navigation.scss

```diff
@@ -100,31 +127,37 @@
       }
 
       .p-navigation__link {
```

Comment:
In some websites we also have external links in the header, should we add a BEM extension class to add this style?